landscape.yml

```diff
@@ -270,8 +270,8 @@ landscape:
           - item:
             name: DockerHub
```

Comment:
```suggestion
            name: Docker Hub
```

Reply:
I think most folx know it is `DockerHub`

Reply:
Oleg is correct, its Docker Hub, not DockerHub. Its not like GitHub :) But i can understand why people might assume so.